src/dapp/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.2] - 2018-12-21
```

Comment:
We should add the actual link at the bottom. In order to to this, the version has to be tagged (we can do this after merging), and the tag pushed.

Reply:
Also, is this a bugfix? I think it's more like a backwards-compatible new feature, and so should bump the minor version, to 0.10.1.

If I read [SemVer](https://semver.org/) correctly.

Reply:
Yep, you're right. I'm used to only modifying the last number, but we should really aim for a 1.0 soon. I'll fix

Reply:
Doesn't it change to `0.10.0` instead of `0.10.1`?

Reply:
Yes, of course, sorry :sweat_smile:

Reply:
Oh and another thing: we don't have to bump the version every time. We can bundle up a few changes under `Unreleased` and only release whenever we want. People will still get the `master` version 99.9% of the times anyway, the changelog is mostly to document what happened, less for actually shipping releases.

Whatever is easiest.